### PR TITLE
Create activationLikelihoodEstimation.jsonld

### PR DESCRIPTION
**Pull request by lzehl**
request via email

### CODE REVIEW

File: instances/technique/activationLikelihoodEstimation.jsonld [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/activationLikelihoodEstimation",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "An 'activation likelihood estimation' is a coordinate-based meta-analysis of neuroimaging data that determines the above-chance convergence of activation probabilities between experiments (i.e., not between foci). [adapted from [Eickhoff et al., 2011](https://dx.doi.org/10.1016%2Fj.neuroimage.2011.09.017)]",
```
Comment (skoehnen) [inline instances/technique/activationLikelihoodEstimation.jsonld:7]: I think here is one bracket too much
`ci). [adapted from`

Comment (skoehnen) [reply instances/technique/activationLikelihoodEstimation.jsonld:7]: Or did you want the link from adapted on, or from Eickhoff.

Comment (lzehl) [reply instances/technique/activationLikelihoodEstimation.jsonld:7]: no that is on purpose. (square brackets around the "adapted from  Eickhoff et al., 2011" where "Eickhoff et al., 2011" is then a link

Comment (lzehl) [reply instances/technique/activationLikelihoodEstimation.jsonld:7]: @skoehnen or did I miss something?

Comment (skoehnen) [reply instances/technique/activationLikelihoodEstimation.jsonld:7]: Okay, just making sure :)

**Review by skoehnen — APPROVED** [review]
Looks good to me

**Pull request merged**